Documentation/ApiOverview/DirectoryStructure/LegacyInstallations.rst

```diff
@@ -12,12 +12,20 @@ The structure below describes the directory structure in a legacy
 TYPO3 installation without Composer. For the structure in a Composer-based installation
 see :ref:`Composer-based installations: Directory structure <directory-structure>`.
 
+.. _typo3-project-root-folder:
+
+TYPO3 project root folder
+=========================
+
+The TYPO3 project root folder is equal to your web root folder or to any subfolder of it, 
+e.g. :file:`/home/user1/public_html/project1/`, where `project1` is the name of the project.
```

Comment:
Please, use a common directory, like `/var/www/html` for Debian-based distributions (or similar for RedHat etc.). I don't think, we should guide users to use a user's home directory for a website.

Reply:
What is the problem with a user's home directory? I think that most TYPO3 users will not have their own servers running, but they are participating on a shared host environment. 
But of course you can change this example as you like.

Reply:
I think we should not use any real path but something like `/path/to/your/webroot`

Reply:
@franzholz it would be great if you could change it to Lina's proposition then the issue can be approved.